origami.json

```diff
@@ -31,19 +30,19 @@
 			"data": {
 				"modifierClass": "o-table--compact"
 			},
-			"expanded": false,
+			"hidden": true,
```

Comment:
We probably shouldn't hide all these demos, I don't think displaying them all is correct either, but haven't had chance to put this through design yet. Maybe we should display all until we can clarify with design which ones should be used so we can hide the rest?